javascript/GridfieldDropdownFilter.js

```diff
@@ -0,0 +1,9 @@
+(function ($) {
+    $.entwine('ss', function ($) {
+        $('.gridfield-dropdown-filter select').entwine({
+            onchange: function() {
+                this.parent().find('.action').click();
```

Comment:
Consider adding a comment noting this is how the core component "filter headers" works...
Then it'll be less of a hack :)

Reply:
👍 Done: 
> Trigger the action when the select is changed. This clicks a hidden button that is entwined by GridField.js . This is similar to how GridFieldFilterHeader works.